go.mod

```diff
@@ -1,33 +1,32 @@
 module github.com/zlabjp/spire-openstack-plugin
 
-go 1.13
+go 1.14
```

Comment:
In addition to this change, I think the following files should be updated:

- https://github.com/zlabjp/spire-openstack-plugin/blob/71c7407ef7125e8e189c385200af15f35bae8730/CONTRIBUTING.md
- https://github.com/zlabjp/spire-openstack-plugin/blob/7e9b05946219a79ab72051f80214340cbea9bd7a/.travis.yml